### PR TITLE
channel support

### PR DESCRIPTION
**Pull request by markaddleman**
core.async channel support for tripod interceptor system.

I had to remove boot.properties file to get boot to run on my system.  I have never used boot before so if there is a more appropriate workaround, I don't know what it is.  

Happy to accept critiques for improving the PR

### CODE REVIEW

File: src/tripod/context.cljc [hunk 1]
```diff
@@ -79,6 +83,16 @@
 #?(:cljs
    (defn with-bindings [_ res] res))
 
+(defn channel? [x]
+  (satisfies? asyncp/ReadPort x))
+
+(declare terminate)
+(declare execute)
+
+(defn go-async [old-context context-ch]
```
Comment (frankiesardo) [inline src/tripod/context.cljc:92]: Shouldn't this be more like

```clj
(defn go-async [old-context context-ch]
  (go 
    (execute (<! context-ch))
    (terminate old-context)))
```

Comment (markaddleman) [reply src/tripod/context.cljc:92]: Yes.  I was playing around with the idea of placing the go channel into the context and only partially removed it.  

New PR on its way

Comment (markaddleman) [reply src/tripod/context.cljc:92]: I guess I don't need a new PR.  Has this one been updated with commit  11c1a3a  ?

**Comment by frankiesardo** [comment]
Thanks! :+1:

**Pull request merged**